source/MaterialXRuntime/Private/PvtPort.h

```diff
@@ -71,72 +71,21 @@ class PvtPort : public PvtObject
         return (_flags & RtPortFlag::SOCKET) != 0;
     }
 
-    const RtIdentifier& getColorSpace() const
```

Comment:
Moved these from RtPort to RtInput class for better separation of concerns.

Reply:
Outputs can have colorspace so I don't think you can move colorspace RtInput only.
Units can be moved as there is no concept of output units.

Reply:
That's interesting, I didn't know that. Had a look in the spec now and you are right, it's there for output texture baking.